docs/reference/data-management.asciidoc

```diff
@@ -0,0 +1,33 @@
+[role="xpack"]
+[[data-management]]
+= Data management
+
+[partintro]
+--
+The data you store in {es} generally falls into one of two categories:
+
+* A collection of content you want to search such as a catalog of products
```

Comment:
I believe the `collection` term is already established/taken in the industry when it comes to content/storing documents. `collection of content` sounds a bit odd to me as `content`, when it comes to storing data, already includes/implies multiple documents. 
I think we should just call it `content`